apps/bmd/src/pages/AdminScreen.tsx

```diff
@@ -22,11 +26,14 @@ import {
 } from '../utils/date'
 import InputGroup from '../components/InputGroup'
 import Modal from '../components/Modal'
+import VersionsData from '../components/VersionsData'
 
 type Meridian = 'AM' | 'PM'
 
 interface Props {
+  // TODO: replace both of these with machineConfig to get machineId too
```

Comment:
This is from my WIP. Seems like you did include `machineConfig` and can remove this?